### DISCUSSION TITLE
Storybook + Dark Mode

### DISCUSSION THREAD

**Discussion by hariria** [chakra-ui/panda#2117] [General]
There aren't any guides for how to configure dark mode with storybook with dark mode provided in `@storybook/addon-essentials` and the default panda configuration. Does anyone have any experience with this and if so, could we add it to the section here https://panda-css.com/docs/installation/storybook?

**Comment by astahmer** [comment]
I'm not familiar with the exact way storybook applies dark mode (could be a class or a data-attribute), but you can use a Panda CSS `condition` (which is just a selector or a media query), either one of the [defaults](https://panda-css.com/docs/concepts/conditional-styles#reference) or create your own https://panda-css.com/docs/customization/conditions
